packages/cashier/src/pages/withdrawal/withdrawal-crypto-receipt/withdrawal-crypto-receipt.tsx

```diff
@@ -126,7 +155,7 @@ const WithdrawalCryptoReceipt = observer(() => {
                 <Localize i18n_default_text='Your withdrawal will be processed within 24 hours' />
             </Text>
             <div className='withdrawal-crypto-receipt__detail'>
-                {!isMobile() && <Status />}
+                {!isMobile() && <Status last_transaction={last_transaction} />}
```

Comment:
I think we can replace this `isMobile()` function with `is_mobile` since you are calling observer in this function